Kuroba/app/build.gradle

```diff
@@ -295,8 +295,10 @@ dependencies {
     implementation "androidx.core:core-ktx:${versions.core_ktx}"
     implementation "androidx.work:work-runtime-ktx:2.4.0"
     implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.2.0"
-    implementation 'com.google.android.material:material:1.2.1'
-    implementation 'com.google.android.exoplayer:exoplayer:2.12.0'
+    implementation "com.google.android.material:material:${versions.material}"
+
+    implementation 'com.google.android.exoplayer:exoplayer-core:2.12.0'
+    implementation 'com.google.android.exoplayer:exoplayer-ui:2.12.0'
```

Comment:
Since there are two separate dependencies for ExoPlayer now could you move these two version numbers into versions.gradle?

Reply:
Sure.